### PR TITLE
update wp8 to 0.0.9 of windows-push

### PR DESCRIPTION
**Pull request by edewit**
fixing issue [AGCORDOVA-120](https://issues.jboss.org/browse/AGCORDOVA-120)

### CODE REVIEW

File: src/wp8/shared/Registration.cs [hunk 1]
```diff
@@ -69,10 +123,13 @@ public abstract class Registration
         protected void OnPushNotification(string message, IDictionary<string, string> data)
         {
             EventHandler<PushReceivedEvent> handler = PushReceivedEvent;
-            CreateChannelStore().Save(PUSH_ID_KEY, data[PUSH_ID_KEY]);
+            if (data != null && data.ContainsKey(PUSH_ID_KEY))
```
Comment (edewit) [inline src/wp8/shared/Registration.cs]: @lfryc the update adds this check

**Comment by lfryc** [comment]
@edewit you did some cleanup, right? :-)

**Comment by lfryc** [comment]
@edewit I'm getting some errors + warning when `cordova run wp8 --device`:

````
Plugins\aerogear-cordova-push\PushPlugin.cs(66,55): error CS1061: 'PushNotifica
tion' does not contain a definition for 'message' and no extension method 'mess
age' accepting a first argument of type 'PushNotification' could be found (are
you missing a using directive or an assembly reference?) [C:\Users\Lukas\worksp
aces\unified-push-helloworld\cordova\platforms\wp8\AeroGear_UnifiedPush_HelloWo
rld.csproj]
Plugins\aerogear-cordova-push\PushPlugin.cs(66,81): error CS1061: 'PushNotifica
tion' does not contain a definition for 'data' and no extension method 'data' a
ccepting a first argument of type 'PushNotification' could be found (are you mi
ssing a using directive or an assembly reference?) [C:\Users\Lukas\workspaces\u
nified-push-helloworld\cordova\platforms\wp8\AeroGear_UnifiedPush_HelloWorld.cs
proj]
Plugins\aerogear-cordova-push\MpnsRegistration.cs(89,48): warning CS1998: This
async method lacks 'await' operators and will run synchronously. Consider using
 the 'await' operator to await non-blocking API calls, or 'await Task.Run(...)'
 to do CPU-bound work on a background thread. [C:\Users\Lukas\workspaces\unifie
d-push-helloworld\cordova\platforms\wp8\AeroGear_UnifiedPush_HelloWorld.csproj]
````

**Comment by lfryc** [comment]
:+1: @edewit works now, merging

Found just a small bug in UPS console during testing this: https://issues.jboss.org/browse/AGPUSH-1524

**Comment by lfryc** [comment]
Landed :rocket: !!

**Comment by lfryc** [comment]
Sry @edewit it took a while before I got to it :-)

**Pull request merged**